test/Maker.spec.js

```diff
@@ -7,7 +7,7 @@ async function createMaker(privateKey) {
 }
 
 test('create without any options', async () => {
-  await Maker.create('test');
+  await Maker.create('test', { log: false });
```

Comment:
please remove this -- the test is specifically checking `create` without a second argument